project-base/src/Controller/Front/OrderController.php

```diff
@@ -421,12 +421,6 @@ private function getTermsAndConditionsResponse(): Response
      */
     private function sendMail(Order $order): void
```

Comment:
I think this method might no longer be necessary now that it contains only 1 line.

Reply:
removed